packages/ui/src/components/ui/color-picker.tsx

```
@@ -0,0 +1,408 @@
+							}
+						/>
+						<span className="font-mono text-fontSize-xs uppercase">{currentHex}</span>
+					</Button>
+				</PopoverTrigger>
+				<PopoverContent className="w-[320px] space-y-3 p-2" align="start">
+					<div
+						ref={squareRef}
+						onMouseDown={onSquareMouseDown}
+						className="relative h-52 cursor-crosshair overflow-hidden rounded-sm border"
+						style={{ backgroundColor: `hsl(${hsv.h} 100% 50%)` }}
+					>
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)